server/ocr_server.cpp

```diff
@@ -0,0 +1,9 @@
+#include "lib/log.hpp"
+#include <unistd.h>
+#include "ocr_server.hpp"
+
+void ocrserver::OcrServer::StartServer(void) {
+  sys::Log log;
+  log.Info("Starting OCR Server\r\n");
+  sleep(10);
```

Comment:
This is a bit longer: std::this_thread::sleep_for(std::chrono::milliseconds(10));

But is portable and is available from C++ 11 onwards.